### PR TITLE
feat: add feature flags to disable RDB courses

### PR DESCRIPTION
**Pull request by moT01**
This adds two new growthbook flags for the RDB courses. A disabling soon, and disabled flag. 

To test: Be sure to add the growthbook URI to your .env, then toggle the development version of those flags in the growthbook web UI.

Alternatively, you can set any of the variables to `{ on: true }` to enable them.

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: client/src/templates/Challenges/codeally/show.tsx [hunk 1]
```diff
@@ -304,14 +304,12 @@ class ShowCodeAlly extends Component<ShowCodeAllyProps> {
...
                   )}
-                <Alert id='codeally-cookie-warning' variant='info'>
-                  <p>{t(`intro:misc-text.enable-cookies`)}</p>
-                </Alert>
                 <Button
                   aria-describedby='codeally-cookie-warning'
                   block={true}
                   variant='primary'
                   data-cy='start-codeally'
+                  disabled={true}
```
Comment (ShaunSHamilton) [inline client/src/templates/Challenges/codeally/show.tsx:312]: This does not really matter much, but a client-side disable does not stop anything - someone can still invoke/access `tryToShowCodeAlly` with a disabled button.

That is to say, we might want to remove the `iframe` altogether? _I do not know why I keep reviewing drafts_ 🤪

Comment (moT01) [reply client/src/templates/Challenges/codeally/show.tsx:281]: ✔️

Comment (moT01) [reply client/src/templates/Challenges/codeally/show.tsx:281]: This PR is meant to just quickly disable the browser version of RDB. I plan to clean up all the code with the PR that enables the gitpod setup.

File: client/i18n/locales/english/intro.json [hunk 2]
```diff
@@ -1448,6 +1448,9 @@
     "legacy-desc": "These courses are no longer part of the certification path, but are still available for you to further your learning.",
     "legacy-go-back": "Go to the current version of the curriculum.",
     "course-maintenance": "These courses are undergoing maintenance. If they are not working, you can learn how to run them locally at <0>https://www.freecodecamp.org/news/how-to-run-freecodecamps-relational-databases-curriculum-using-docker-vscode-and-coderoad</0>.",
+    "course-disabling-soon": "The browser version of these courses will be temporary disabled soon and your virtual machines will be deleted. Any progress in your virtual machines will be lost. If you have any files you want from them, you should save them to your computer. We apologize for any inconvenience. We hope to have an improved browser version of these courses available again in the next few weeks.",
```
Comment (ShaunSHamilton) [inline client/i18n/locales/english/intro.json:1451]:
```suggestion
    "course-disabling-soon": "The browser version of these courses will be temporarily disabled soon and your virtual machines will be deleted. Any progress in your virtual machines will be lost. If you have any files you want from them, you should save them to your computer. We apologize for any inconvenience. We hope to have an improved browser version of these courses available again in the next few weeks.",
```

File: client/i18n/locales/english/intro.json [hunk 3]
```diff
@@ -1448,6 +1448,9 @@
...
+    "course-disabled": "These courses are temporarily unavailable to run in the browser. We apologies for any inconvenience. You can learn how to run them locally at <0>https://www.freecodecamp.org/news/how-to-run-freecodecamps-relational-databases-curriculum-using-docker-vscode-and-coderoad</0>. We hope to have an improved browser version available again in the next few weeks.",
```
Comment (ShaunSHamilton) [inline client/i18n/locales/english/intro.json:1452]:
```suggestion
    "course-disabled": "These courses are temporarily unavailable to run in the browser. We apologize for any inconvenience. You can learn how to run them locally at <0>https://www.freecodecamp.org/news/how-to-run-freecodecamps-relational-databases-curriculum-using-docker-vscode-and-coderoad</0>. We hope to have an improved browser version available again in the next few weeks.",
```

**Review by ShaunSHamilton — DISMISSED** [review]
Everything else LGTM 👍

**Comment by ojeytonwilliams** [comment]
@moT01 splitting each alert into its own component makes it a bit more readable (to me, at least). The main component becomes

```tsx
{
  const codeAllyDownFeature = useFeature('codeally_down');
  const codeAllyDisablingSoonFeature = useFeature('codeally_disabling_soon');
  const codeAllyDisabledFeature = useFeature('codeally_disabled');

  return codeAllyDisabledFeature.on ? (
    <Disabled />
  ) : codeAllyDisablingSoonFeature.on ? (
    <DisablingSoon />
  ) : codeAllyDownFeature.on ? (
    <Down />
  ) : null;
}
```

which seems quite nice.

 Mind if I push a commit?

**Comment by moT01** [comment]
Thanks for taking a look @ShaunSHamilton @ojeytonwilliams 👍 yea, you can push to this if you want.

**Review by ojeytonwilliams — APPROVED** [review]
Sorry for the delay, Tom. Anyways, it all LGTM :+1:

**Review by raisedadead — APPROVED** [review]

**Pull request merged**